pkg/virt-controller/watch/vm.go

```diff
@@ -967,10 +1080,10 @@ func (c *VMController) getVirtualMachineBaseName(vm *virtv1.VirtualMachine) stri
 func (c *VMController) processFailure(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance, createErr error) {
 	reason := ""
 	message := ""
-	log.Log.Object(vm).V(4).Infof("Processing failure status:: shouldRun: %t; noErr: %t; noVm: %t", vm.Spec.Running, createErr != nil, vmi != nil)
+	log.Log.Object(vm).V(4).Infof("Processing failure status:: shouldRun: %v; noErr: %t; noVm: %t", vm.Spec.Running, createErr != nil, vmi != nil)
```

Comment:
Now that you mention it, I removed the reference to "Running" because in this context it's a pointer, which is unhelpful gibberish in a log message. Replaced it with vm.RunStrategy() which will always have something useful and readable.